addons/uav/CfgVehicles.hpp

```diff
@@ -0,0 +1,388 @@
+class CBA_Extended_EventHandlers;
+
+
+class Turrets;
+class MainTurret;
+class AnimationSources;
```

Comment:
That's completely wrong.

Reply:
Referenced material from here:
https://forums.bohemia.net/forums/topic/170779-set-gun-elevation/?tab=comments#comment-2671993

Reply:
There is no "less wrong" here. You either get the config right, or you break shit. This breaks shit. So does whatever is posted on the BIF (can't review, cause 504 error, don't really care either).

Reply:
There is no AnimationSources at root level